src/components/interactive-builder/modals/question/question-form/question/question.component.tsx

```diff
@@ -31,6 +32,12 @@ const Question: React.FC<QuestionProps> = ({ checkIfQuestionIdExists }) => {
     return checkIfQuestionIdExists(formField.id);
   }, [formField.id, checkIfQuestionIdExists]);
 
+  const handleQuestionInfoLabel = () => {
+    const { questionInfo, ...updatedFormField } = formField;
+    setFormField(updatedFormField);
+    setShowQuestionInfoInput(false);
+  };
```

Comment:
Please wrap this in a `useCallback` for better performance. When doing so, also ensure that both `formField` and `setFormField` are not dependencies since `setFormField` updates `formField`. Here's a place where its done similarly for guidance - https://github.com/openmrs/openmrs-esm-form-builder/blob/d80c22868c17a39df4838563947eeed81c9f9ffb/src/components/interactive-builder/modals/question/question-form/question-types/inputs/obs/obs-type-question.component.tsx#L31